app/src/renderer/components/network/PageBlock.vue

```diff
@@ -113,10 +141,11 @@ export default {
         `queryBlockInfo`,
         $route.params.height
       )
+      await $store.dispatch(`getBlockTxs`, $route.params.height)
 
       if (
         !blockInfo &&
-        Number($route.params.height) > Number(lastHeader.height)
+				Number($route.params.height) > Number(lastHeader.height)
```

Comment:
not sure about this spacing

Reply:
yeah in theory the && should be in the newline, although I don't like it too much

Reply:
also tabspace is off. 1 tab = 2 spaces.

Reply:
> 1 tab = 2 spaces

I have that on my settings

Reply:
but it is 4 spaces here.

Reply:
got it, I have to add the rule to reject any tab